etl-pipeline/managers/s3.py

```diff
@@ -26,15 +26,19 @@ def __init__(self):
             endpoint_url=os.environ.get('S3_ENDPOINT_URL', None)
         )
 
-    def store_pdf_manifest(self, record: Record, bucket_name, flags=FileFlags(reader=True), path: str=None):
+    def store_pdf_manifest(self, record: Record, bucket_name):
         record_id = record.source_id.split('|')[0]
         pdf_part = next(filter(lambda part: part.file_type == 'application/pdf', record.parts), None)
+        manifest_part = next(filter(lambda part: part.file_type == 'application/webpub+json', record.parts), None)
+
+        if manifest_part is not None and pdf_part is not None:
+            manifest_json = self.generate_manifest(record=record, source_url=pdf_part.url, manifest_url=manifest_part.url)
+            self.create_manifest_in_s3(manifest_path=manifest_part.file_key, manifest_json=manifest_json, bucket=bucket_name)
+
+            return
 
         if pdf_part is not None:
-            if path:
-                manifest_path = f'manifests/{path}/{record.source}/{record_id}.json'
-            else:
-                manifest_path = f'manifests/{record.source}/{record_id}.json'
```

Comment:
Just to check - the existing thing puts these under the 'publisher_backlist' folder - are there any implications to no longer doing that?

Reply:
I think the logic is the same - i just add the manifest part in the mapper with the proper path and flags!

Reply:
Hmmm, I guess I'm just confused because of the removal of the `path` parameter from this method and from the manifest path in the line below.